server/controllers/finance/reports/journal/index.js

```diff
@@ -32,15 +33,64 @@ function postingJournalExport(req, res, next) {
   });
 
   let report;
-
   try {
     report = new ReportManager(REPORT_TEMPLATE, req.session, options);
   } catch (e) {
     return next(e);
   }
 
-  const unposted = Journal.buildTransactionQuery(options, false);
-  const posted = Journal.buildTransactionQuery(options, true);
+  return Journal.journalEntryList(options)
+    .then(rows => report.render({ rows }))
+    .then((result) => {
+      if (result.headers.type === 'xlsx') {
+        res.xls(result.headers.filename, filter(result.report.rows));
+      } else {
+        res.set(result.headers).send(result.report);
+      }
+    })
+    .catch(next)
+    .done();
+}
+
+function dateFormat(date) {
```

Comment:
Can you please combine this function with the date format function in the CSV renderer?  It will make it much easier to maintain.  This was mentioned in a previous review.